docs/sources/mimir/operators-guide/monitor-grafana-mimir/installing-dashboards-and-alerts.md

```diff
@@ -67,3 +67,95 @@ Given the exact setup really depends on a case-by-case basis, the following inst
      },
    }
    ```
+
+### Deploy mixin with terraform
+
+Based on the earlier step, you can use [Terraform](https://www.terraform.io/) to deploy the mixin in both Grafana and Grafana Mimir. To do so you can use both [grafana/grafana](https://registry.terraform.io/providers/grafana/grafana/latest), [ovh/mixtool](https://registry.terraform.io/providers/ovh/mixtool/latest) and [ovh/mimirtool](https://registry.terraform.io/providers/ovh/mimirtool/latest) providers.
```

Comment:
```suggestion
Based on the earlier step, you can use [Terraform](https://www.terraform.io/) to deploy the mixin in both Grafana and Grafana Mimir. To do so, you can use both [grafana/grafana](https://registry.terraform.io/providers/grafana/grafana/latest), [ovh/mixtool](https://registry.terraform.io/providers/ovh/mixtool/latest), and [ovh/mimirtool](https://registry.terraform.io/providers/ovh/mimirtool/latest) providers.
```
```suggestion
Based on the earlier step, you can use [Terraform](https://www.terraform.io/) to deploy the mixin in both Grafana and Grafana Mimir. To do so, you can use the [grafana/grafana](https://registry.terraform.io/providers/grafana/grafana/latest), [ovh/mixtool](https://registry.terraform.io/providers/ovh/mixtool/latest), and [ovh/mimirtool](https://registry.terraform.io/providers/ovh/mimirtool/latest) providers.
```
This task should be as atomic as possible. What does "Based on the earlier step" mean?

I removed "both" because there were three items. Please vet my change for technical accuracy.

Reply:
"Based on the earlier step" refer to the previous paragraph "Install dashboards from Jsonnet mixin". It could have been better phrased like "Based on Jsonnet configuration file"